dataland-frontend/src/components/resources/newLandingPage/emailButton.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
File name should start with capital "E"

Reply:
renamed in EmailButton.vue

Reply:
Somehow it seems that the renaming didn't work.  It is still with a small "e":

![image](https://github.com/d-fine/Dataland/assets/99255296/ec80ba0a-b052-460c-9c0f-f99ed2a18449)

Reply:
It's weird, the file is renamed but somehow  was ignored and not added to the commit. I'll try to fix it

Reply:
It is correct now. Thx